onnx_coreml/_mlmodel_pass.py

```diff
@@ -0,0 +1,61 @@
+def _get_nn_spec(spec):
+    if spec.WhichOneof('Type') == 'neuralNetwork':
+        nn_spec = spec.neuralNetwork
+    elif spec.WhichOneof('Type') == 'neuralNetworkClassifier':
+        nn_spec = spec.neuralNetworkClassifier
+    elif spec.WhichOneof('Type') == 'neuralNetworkRegressor':
+        nn_spec = spec.neuralNetworkRegressor
+    else:
+        raise ValueError('Specification must contain a neural network')
+    return nn_spec
+
+def _dead_code_elimination(spec):
+    """
+    dead_code_elimination: Remove unused layers from Specification
```

Comment:
can you elaborate the comment here, that is, how are you defining unused? 
Layers whose output(s) is not getting consumed by any other layer, nor is it an overall model output?

Reply:
Not used by any of the layer.
We are not removing the output.

I thought of not updating Output as it's a handshake with the model being converted and we should preserve the input and output